tests/test_csv_reader.py

```diff
@@ -90,7 +90,7 @@ def test_extract_snex_metadata(test_config, snowex_csv):
 
 
 def test_extract_generic_metadata(test_config, generic_csv_content, generic_csv):
-    metadata = csv_reader.extract_metadata(generic_csv, test_config)
+    metadata = csv_reader.extract_metadata(generic_csv, None, test_config)
```

Comment:
I wonder if we could combine these two unit tests so that the parametrization (sp? word?) includes the two csv readers. I haven't thought about it, so I'm not sure if this is difficult or not.

Reply:
> I wonder if we could combine these two unit tests so that the parametrization (sp? word?) includes the two csv readers. I haven't thought about it, so I'm not sure if this is difficult or not.

I think we could, although I'm a little reluctant to mess with it at the moment. I'm sure I'll get the urge to tackle unrelated cleanup in the next story or two!